velox/functions/prestosql/GreatestLeast.cpp

```diff
@@ -193,7 +198,7 @@ VELOX_DECLARE_VECTOR_FUNCTION(
 
 VELOX_DECLARE_VECTOR_FUNCTION(
     udf_greatest,
-    GreatestFunction ::signatures(),
```

Comment:
Does this mean `signatures` was never being passed correctly?

Reply:
No. I just removed redundant space.